package.json

```diff
@@ -179,6 +180,7 @@
     "jest-canvas-mock": "^2.4.0",
     "jest-environment-jsdom": "^27.5.1",
     "jest-localstorage-mock": "^2.4.22",
+    "lokijs": "^1.5.12",
```

Comment:
where is this being used?

Reply:
both being used in a unit test, because i couldn't get the instanceof check to pass in useIsSafe, there might be a better way though

Reply:
ok wasn't that hard to fix, can remove these too